docs/cli/usage/customcf.md

```diff
@@ -1,18 +1,24 @@
 ---
-title: Custom AWS Resources
+title: Add custom AWS resources
 description: The Amplify CLI provides escape hatches for modifying the backend configurations generated in the form of CloudFormation templates by the CLI. This allows you to use the CLI for common flows but also any advanced scenarios which aren't provided in the standard category workflows.
 ---
 
-The Amplify CLI provides escape hatches for modifying the backend configurations generated in the form of CloudFormation templates by the CLI. This allows you to use the CLI for common flows but also any advanced scenarios which aren't provided in the standard category workflows.
+The Amplify CLI provides escape hatches for modifying the backend configurations generated in the form of CloudFormation templates by the CLI. 
 
-* For your GraphQL API generated by the Amplify CLI, you have the ability to [override resolvers](~/cli/graphql-transformer/resolvers.md#overwriting-resolvers) as well as add your own [custom resolvers](~/cli/graphql-transformer/resolvers.md#custom-resolvers) to add on to the CLI generated CloudFormation stacks.
+## How to extend Amplify CLI
 
-* For majority of the other categories, you can locally edit and manage the CloudFormation file generated by the CLI in the `amplify/backend/<category>/<cloudformation-template.json/yml>` location. [Read more about Cloudformation](https://aws.amazon.com/cloudformation/)
+The most common scenarios for extending your app beyond Amplify CLI's features are to:
+* customize your GraphQL API business logic by [overriding resolvers](~/cli/graphql-transformer/resolvers.md#overwriting-resolvers) or author your own [custom resolvers](~/cli/graphql-transformer/resolvers.md#custom-resolvers).
+* override Amplify CLI's generated CloudFormation by editing `amplify/backend/<category>/<cloudformation-template.json/yml>`. [Read more about Cloudformation](https://aws.amazon.com/cloudformation/)
```

Comment:
Maybe we can give an example of "Override Amplify CLI's generated CloudFormation" like maybe "like updating the node.js version of a Lambda function" or "like updating the base configuration of an Amplify generated service" just to give more insight as to what this would look like.